engine/apps/api/urls.py

```diff
@@ -69,7 +72,7 @@
 router.register(r"oncall_shifts", OnCallShiftView, basename="oncall_shifts")
 
 urlpatterns = [
-    path("", include(router.urls)),
+    # path("", include(router.urls)), # TODO: this can be removed
```

Comment:
Why it can be removed? Isn't it required to serve routers registered in router? https://www.django-rest-framework.org/api-guide/routers/

Reply:
because it is also done on line 
https://github.com/grafana/oncall/pull/2261/files#diff-d87ee8df9c3c7eed2c4b729461f3831c5318b69907c58cdc0a2e566c170c2e9bL119